### PR TITLE
GSLUX-670: Fix drag and drop for firefox

### PR DESCRIPTION
**Pull request by AlitaBernachot**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-XXX

### Description

- Fixed drag and drop on firefox (force Sortablejs to enable dragndrop fallback -no html5 dnd-)
- Fixed ghost dragging style for firefox
- Added `--force` option to force vite to clean node_modules cache on `npm start` (dev mode only)

### CODE REVIEW

**Review by mki-c2c — CHANGES_REQUESTED** [review]
Integration in v3 does not seem to work correctly:
while dragging, all layers become invisible and items can only be dragged upwards, not downwards.

**Review by mki-c2c — APPROVED** [review]
could you please rename the typos ?

File: src/components/layer-manager/layer-item/layer-item.spec.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ describe('LayerItem', () => {
       props: {
         is3d: false,
         layer: layerMock,
-        draggableClassName: 'classnamedragg',
+        dragHandlerClassName: 'classnamedragg',
```
[the remark this answers is not in the document]
Comment (AlitaBernachot) [reply src/components/layer-manager/layer-item/layer-item.spec.ts]: yes, good point

**Comment by AlitaBernachot** [comment]
> could you please rename the typos ?

yes, done 🙂

**Pull request merged**